sdk_blueprints/Gretel_101_Blueprint.ipynb

```diff
@@ -77,169 +67,197 @@
                 "\n",
                 "# explore the data using pandas\n",
                 "df = pd.read_csv(dataset)\n",
                 "df.head()"
-            ],
-            "metadata": {
-                "id": "cW3VKpyPvm6W"
-            },
+            ]
+        },
+        {
+            "cell_type": "code",
             "execution_count": null,
-            "outputs": []
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# Patient dataset is sequential which is grouped based on \"patient_id\". Each group is ordered by the \"even_id\".\n",
+                "df_sorted = df.groupby('patient_id', group_keys=True).apply(lambda group: group.sort_values('event_id'))\n",
+                "df_sorted "
+            ]
         },
         {
             "cell_type": "markdown",
+            "metadata": {
+                "id": "SwROZthrvXil"
+            },
             "source": [
                 "## 🏋️‍♂️ Train a generative model\n",
                 "\n",
-                "- The [tabular-actgan](https://github.com/gretelai/gretel-blueprints/blob/main/config_templates/gretel/synthetics/tabular-actgan.yml) base config tells Gretel which model to train and how to configure it.\n",
+                "- The [navigator-ft](https://github.com/gretelai/gretel-blueprints/blob/main/config_templates/gretel/synthetics/navigator-ft.yml) base config tells Gretel which model to train and how to configure it.\n",
                 "\n",
-                "- You can replace `tabular-actgan` with the path to a custom config file, or you can select any of the tabular configs [listed here](https://github.com/gretelai/gretel-blueprints/tree/main/config_templates/gretel/synthetics).\n",
+                "- You can replace `navigator-ft` with the path to a custom config file, or you can select any of the tabular configs [listed here](https://github.com/gretelai/gretel-blueprints/tree/main/config_templates/gretel/synthetics).\n",
                 "\n",
                 "- The training data is passed in using the `data_source` argument. Its type can be a file path or `DataFrame`.\n",
                 "\n",
                 "- **Tip:** Click the printed Console URL to monitor your job's progress in the Gretel Console."
-            ],
-            "metadata": {
-                "id": "SwROZthrvXil"
-            }
+            ]
         },
         {
             "cell_type": "code",
-            "source": [
-                "trained = gretel.submit_train(\"tabular-actgan\", data_source=dataset)"
-            ],
+            "execution_count": null,
             "metadata": {
                 "id": "i89eGZwIxSCW"
             },
-            "execution_count": null,
-            "outputs": []
+            "outputs": [],
+            "source": [
+                "# For car accident data, the \"group_training_examples_by\" and \"order_training_examples_by\" params can be dropped since the data is not sequantial.\n",
```

Comment:
suggestion: Requiring the user to make code changes after selecting the dataset is not great. We should either do that automatically for them, or not use groupby as mentioned above.